relay/sources/rudder-server-relay-postinst

```diff
@@ -105,6 +106,33 @@ fi
 sed -i '/^Set up a WSGI serving process/d' /etc/${APACHE_VHOSTDIR}/rudder.conf
 sed -i '/^WSGI/d' /etc/${APACHE_VHOSTDIR}/rudder.conf
 
+# Apply SELinux config before starting the services
+if [ "${SELINUX}" -eq 1 ]; then
+  # Check "sestatus" presence, if not there there is not SELinux
+  # setup on this system
+  if type sestatus >/dev/null 2>&1 && sestatus | grep -q "enabled"; then
+    # Add/Update the rudder-relay SELinux policy
+    semodule -i /opt/rudder/share/selinux/rudder-relay.pp
+    # Ensure inventory directories context is set by resetting
+    # their context to the contexts defined in SELinux configuration,
+    # including the file contexts defined in the rudder-relay module
+    restorecon -R /var/rudder/inventories
```

Comment:
is there a way to detect that this changed ? restorecon on /var/rudder/inventories can be painfully slow